renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
Can you remove this unrelated file? Thanks!

Reply:
Removed.